### PR TITLE
Bold the selected header tab text second attempt without affecting wi…

### PR DESCRIPTION
**Pull request by claireoe**
Bold the selected header tab text second attempt without affecting width of header tab

**Issue:**
 Bold the selected header tab text #2263

**Changes:** 
Updated the theme with text shadow to bold selected header tab text.

Validation
home bold
 Dimensions before and after bold applied: 307.39 x48

### CODE REVIEW

**Comment by nguyentvan7** [comment]
image
This doesnt really look right

**Review by frzyc — CHANGES_REQUESTED** [review]
Yeah, I tried this solution before, and I am not too keen on it, since it makes the font sort of "rounded" but I guess this is probably the easiest way to do this. 

Lets iterate on this solution:
- decrease the spread of the textshadow a little, maybe to 0.25px? see if that makes the font less rounded when "bolded"?
- Make the main Tab "Genshin Optimizer" not bolded by default so it does not get double bolded
- Apply the same bolding to the hamburger menu items in mobile view.
- FInally, to pass the format CI, run `yarn run mini-ci`

File: apps/frontend/src/app/Header.tsx [hunk 1]
```diff
@@ -341,12 +342,16 @@ function MobileHeader({
...
                 left: 0,
                 width: '4px',
+
               },
             }}
             sx={(theme) => ({
               '& .Mui-selected': {
                 backgroundImage: `linear-gradient(to right, ${theme.palette.brandGO500.main}, ${theme.palette.neutral700.main})`,
                 color: `${theme.palette.neutral100.main} !important`,
+                // fontWeight: 'Bold',
```
Comment (frzyc) [inline apps/frontend/src/app/Header.tsx:352]: remove

File: apps/frontend/src/app/Header.tsx [hunk 2]
```diff
@@ -341,12 +342,16 @@ function MobileHeader({
               sx: {
                 left: 0,
                 width: '4px',
+
```
Comment (frzyc) [inline apps/frontend/src/app/Header.tsx:345]: remove extra line

File: apps/frontend/src/app/Header.tsx [hunk 3]
```diff
@@ -368,7 +373,7 @@ function MobileHeader({
               }
               iconPosition="start"
               label={
-                <Typography>
+                <Typography >
```
Comment (frzyc) [inline apps/frontend/src/app/Header.tsx:376]: extra space (which is probably failing the format CI)

**Review by frzyc — CHANGES_REQUESTED** [review]
This iteration looks good enough to me. 
Just need to pass format CI, please run `nx format:write`.

**Review by frzyc — APPROVED** [review]
Thanks for iterating on this solution with me. I appreciate the contribution.

**Comment by claireoe** [comment]
@frzyc thanks for the opportunity, enjoyed working on it.

**Pull request merged**